docs/docusaurus/docs/core/define_expectations/test_an_expectation.md

```diff
@@ -131,6 +130,11 @@ Data can be validated against individual Expectations.  This workflow is general
    }
    ```
 
+<!--    When an Expectation uses an Expectation Parameter dictionary you don't have to modify anything on the Expectation object.  Instead, update the dictionary with new values and then test it with the updated dictionary:
```

Comment:
Is this commented out, or is this syntax something else in markdown?

Reply:
It's commented out.  I confirmed with Bill earlier today that we can't currently test individual Expectation Parameter using Expectations like this, but there is a ticket for implementing that post 1.0, so I decided to leave the content there but not rendered for the site.  When the implementation is in place we can just un-comment this.

Reply:
I'm not opposed to removing it entirely, if you think that would be preferable.  If I do I'd probably look up the ticket for adding that functionality and copy the snippets into a comment on it.

Reply:
Totally your call!